docs/tables/aws_guardduty_publishing_destination.md

```diff
@@ -0,0 +1,100 @@
+# Table: aws_guardduty_publishing_destination
+
+AWS Guardduty Publishing Destination provides a resource to export the guard duty findings. This requires an existing GuardDuty Detector.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  detector_id,
+  destination_id,
+  arn,
+  destination_arn,
+  status
+from
+  aws_guardduty_publishing_destination;
+```
+
+### List unverified publishing destination
```

Comment:
```suggestion
### List unverified publishing destinations
```

Please remember to use plural when the query lists multiple resources, for more guidelines please see https://steampipe.io/docs/develop/table-docs-standards#style-conventions-1